content/docs/guides/ads_on_amp/ads_getting_started.md

```diff
@@ -0,0 +1,63 @@
+---
+$title : "Getting Started"
+$order : 0
+$parent: /content/docs/guides/ads_on_amp
+---
+
+## Three Easy Steps to Serving Ads on Your AMP Page
+
+Not sure how to get started? In this tutorial, you’ll learn how to quickly and easily serve ads on your AMP page.
+
+### 1. Add the `<amp-ads>` component to your AMP page:
+
+[sourcecode:html]
+<script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+[/sourcecode]
+
+### 2. You added the ads framework, now specify the ad server or ad network in the `type` attribute:
```

Comment:
I would recommend the steps are started with imperatives and the supplemental information is a sentence or fragment in the step.  For example:

In Step 1:

`### 1. Add the `<amp-ads>` component to your AMP page`

<code snippet>

By adding the `<amp-ads>` component, you've added the AMP ads framework to your page.

In Step 2:

`### 2. Specify the ad server or ad network in the `type` attribute.`

...and so on.  Step 3: Define the ad size.